### PR TITLE
Improve pythran dist

### PR DESCRIPTION
**Pull request by diorcety**
The current implementation run multiple times the transpilation (for example sdist then for build and for install)
This one do that in simpliest way

### CODE REVIEW

**Comment by diorcety** [comment]
Okay the Travis failures don't seem related to my PR

File: pythran/dist.py [hunk 1]
```diff
@@ -9,13 +9,13 @@
 import os.path
 import os
 
-from distutils.command.build_ext import build_ext as LegacyBuildExt
+from setuptools.command.build_ext import build_ext as LegacyBuildExt
```
Comment (serge-sans-paille) [inline pythran/dist.py]: this adds a dependency on setuptools... Is that required ?

**Comment by serge-sans-paille** [comment]
Thans @diorcety for the PR!

**Comment by serge-sans-paille** [comment]
Thanks @diorcety for the PR!

**Pull request merged**